client/src/views/Settings/Tabs/EditProfile/index.vue

```diff
@@ -145,47 +256,115 @@ export default {
   },
 
   created () {
-    this.$store.dispatch('uploadProfile')
+    this.$store.dispatch('profile/getMyProfile')
   },
 
   methods: {
-    createValue (val, done) {
-      if (val.length > 2) {
-        if (!suggestedInterests.includes(val)) {
-          done(val, 'add-unique')
-        }
+    uploadPhotoUrl (val) {
+      if (val.target.files[0].size > 2097152) {
```

Comment:
Don't use magic numbers. Declare intuitive variable why you need exactly this number value.